.github/workflows/on-pullrequest-bidi.yml

```diff
@@ -0,0 +1,101 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: "On Pull Request - Bidi Scanner"
```

Comment:
Linting and testing for the new Bidi scanner code. There are three projects in the bidi-scanner directory, and consequently three jobs.

Initially I created the new scanner as a single command-line tool. After I had that working nicely, I started thinking about how to integrate into GitHub Actions. I realized that I would have to run this as a Docker action. I had some code for changing the logs depending on whether running in GitHub or on own computer. Decided to try something different: creating as a Javascript action.

Handling the differences between GitHub and command line became somewhat tedious, especially when I realized that a JavaScript action must be self-contained: there is no ability to run `npm install`. Thus you must commit into Git all of the dependencies.

To simplify the code (branching on "is this GitHub or not?") and to reduce the size of the dependencies, I split into two interfaces (`cli` and `github`) with some shared code in between them (`lib`). 

Note: only the `cli` package has tests in it.